model/Export/QTIPackedItemExporter.php

```diff
@@ -199,11 +199,14 @@ public function exportManifest($options = [], $exportResult = [])
             }
 
             $manifest = $this->getManifest();
-            $this->getMetadataExporter()->export($this->getItem(), $manifest);
-            $this->injectMetadataToManifest($manifest, $this->getItem());
-            $this->setManifest($manifest);
 
+            //APIP does not support metadata at this moment
+            if (!isset($options['apip']) || (isset($options['apip']) && $options['apip'] !== true)) {
```

Comment:
With an OR, the evaluation of the second member is done only if the first member is false, which means here if $options['apip'] is set. So isset($options['apip']) is useless here. It can be reduced to:
if (!isset($options['apip']) || $options['apip'] !== true)